java/README.md

```diff
@@ -0,0 +1,19 @@
+## Pack64 Java
+
+    A Java version of the Pack64 library to encode and decode vectors into kind-of base64 format.
+
+### To build
+
+    mvn package
+
+Maven will place the jar file in targets/Pack64-1.0.jar
+
+### To run unit tests
+
+    mvn tests
```

Comment:
This should be `mvn test`, to match the target name in the `pom.xml`

Reply:
fixed. good find.